src/components/OcTextarea.vue

```diff
@@ -158,9 +168,10 @@ export default {
       this.$emit("focus", value)
     },
     onKeyDown(e) {
-      if (e.keyCode === 13) {
+      if (this.submitOnEnter && e.keyCode === 13 && !e.ctrlKey && !e.shiftKey) {
```

Comment:
```suggestion
      if (this.submitOnEnter && e.enterKey && !e.ctrlKey && !e.shiftKey) {
```
I like that you're using named keys. Please also change the `keyCode === 13` to the named enterKey then for consistency.

Reply:
sadly keyenter doesn't exist on the event, but I created a mini workaround to make it more readable